README.md

```diff
@@ -71,6 +71,13 @@ curl -Lo simple-ec2 https://github.com/awslabs/aws-simple-ec2-cli/releases/downl
 
 ## Examples
 
+### Connect
```

Comment:
Sorry for all the nit-picking. Can you rename this to `Version`?

Reply:
It is i who has to apologize for being this careless, thanks for your patience. Made the requested change